### ISSUE TITLE
Conflict with RailsAdmin

### ISSUE THREAD

**Issue by jerefrer** [francesc/rails-translate-routes#9]
I had this :

```
mount RailsAdmin::Engine => '/admin', :as => 'rails_admin'
```

near the top of my routes.rb file and everything went just fine until I installed rails-translate-routes.

Then after installing the gem some of my articles#show links turned into "/admin?action=show&controller=articles&id=XX"

An easy workaround was to move the mount RailsAdmin line to the end of the routes.draw do end block.

I don't know if the issue is really an issue, but maybe this will save some time to the next guy encoutering this :)

**Comment by jerefrer** [comment]
Well the workaround is not that easy actually ... Moving the mount line did fix the links issue but the admin becomes unreachable. When I try to load "/admin" I get this error :

```
NoMethodError in Rails_admin/main#dashboard

Showing /home/jeremy/.rvm/gems/ruby-1.8.7-p357/bundler/gems/rails_admin-328d54d0adc5/app/views/rails_admin/main/dashboard.html.haml where line #11 raised:

undefined method `rails_admin_path' for #<Module:0x7f58119e5898>
Extracted source (around line #11):

8:   %tbody
9:     - @abstract_models.each do |abstract_model|
10:       - if authorized? :index, abstract_model
11:         - index_path = index_path(:model_name => abstract_model.to_param)
12:         - row_class = "#{cycle("odd", "even")}#{" link" if index_path}"
13: 
14:         %tr{:class => row_class, :"data-link" => index_path}
```

What do you think ?

**Comment by francesc** [comment]
You probably don't need backend routes translated, try to define the routes this way:

  App::Application.routes.draw do
    ... your regular frontend routes  
  end

  ActionDispatch::Routing::Translator.translate_from_file('config/locales/routes.yml')

  App::Application.routes.draw do
    mount RailsAdmin::Engine => '/admin', :as => 'rails_admin'
  end

**Comment by jerefrer** [comment]
Thanks for your quick reply :)

I tried what you mentionned and actually **it's working** ...

... as long as the routes ordering allows you to do that. And it's not actually my case since I'm using some weird routes and putting the RailsAdmin mount line at the end breaks the hole admin section.

Maybe you could add a _not_localized_ block I could put the mount line in, so it could be anywhere in the routes and I can keep my ordering logic ?

What do you think ?

**Comment by jerefrer** [comment]
I just tried upgrading to Rails 3.2 since the route system has changed a bit, thinking it may solve the issue.

Actually it did a bit since the Admin section is now half-working. Some pages are still not working yet. And also I had to manually add the rails_admin.fr.yml file to my locales as I got a lot of "missing translation" messages all over the admin section. Before using your gem it seemed that the RailsAdmin gem took care of the i18n by itself.

At least I can tell your gem works well with 3.2 :)
